src/main/java/edu/cornell/kfs/module/purap/service/impl/CuElectronicInvoiceHelperServiceImpl.java

```diff
@@ -1289,6 +1283,16 @@ protected void rejectElectronicInvoiceFile(ElectronicInvoiceLoad eInvoiceLoad, S
         }
     }
 
+    private void clearErrorMessagesToPreventRiceSaveAndRoutErrors() {
```

Comment:
Minor typo in method name (should be "...AndRouteErrors" instead of "...AndRoutErrors").

Reply:
thanks